src/include/miopen/solver_id.hpp

```diff
@@ -63,7 +63,7 @@ enum class Primitive
     ReLU,
     Kthvalue,
     SoftMarginLoss,
-    MultiMarginLoss
+    MultiMarginLoss,
```

Comment:
remove `,` if possible to avoid changing unnecessary files

Reply:
its of course in need to separate elements in {}

Reply:
resolved